public/i18n/cn/es6.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>io.js - ES6 on io.js</title>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <meta name="description" content="iojs">
+  <meta name="keywords" content="iojs, io.js, io js, javascript io, uv, libuv, node-forward, node forward, node, node.js, node.js forward, nodejs, nodejs forward, javascript">
+  <link href="https://fonts.googleapis.com/css?family=Lato:400,700" rel="stylesheet">
+  <link href="style.css" rel="stylesheet">
+</head>
+
+  <body>
+
+  <header>
+    <div class="content">
+      <a href="/" class="logo">io.js</a>
+      <div class="spacer"></div>
+      <a href="faq.html">常见问题</a>
+      <a href="es6.html">ES6</a>
+      <a href="https://iojs.org/api/">API 文档</a>
+    </div>
+  </header>
+
+  <div class="content">
+
+    <h1>io.js 运行 ES6</h1>
+
+    <div class="description">
+
+      <p>io.js 是针对 <a href="https://code.google.com/p/v8/">V8</a>
```

Comment:
@coderhaoxin move here https://github.com/xudafeng/website/pull/1/files for Chinese review